### PR TITLE
Add integration-test code coverage

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Will be following up with adding a way to merge all the coverage reports together for a comprehensive coverage report. 

Moved tests under test directory to keep in line with the test folder structure we use across the rest of the product.

### CODE REVIEW

File: extensions/mssql/yarn.lock [hunk 1]
```diff
@@ -2,6 +2,186 @@
 # yarn lockfile v1
 
 
+"@babel/code-frame@^7.10.1":
```
Comment (chlafreniere) [inline extensions/mssql/yarn.lock:5]: why are there so many differences in this yarn.lock file

File: extensions/integration-tests/src/test/cms.test.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import 'mocha';
 import * as vscode from 'vscode';
 import * as azdata from 'azdata';
-import * as mssql from '../../../mssql';
+import * as mssql from '../../../mssql/src/mssql';
```
Comment (chlafreniere) [inline extensions/integration-tests/src/test/cms.test.ts:9]: can you explain this change a bit

Comment (Charles-Gagnon) [reply extensions/integration-tests/src/test/cms.test.ts:9]: This was just being more explicit with the path - but it doesn't need to change (I was changing other things and just didn't revert this). I'll change it back.

File: extensions/mssql/yarn.lock (hunk 1 above)
Comment (Charles-Gagnon) [reply extensions/mssql/yarn.lock:5]: So for some reason the yarn.lock was referencing the 1.0.5 version of vscodetestcover - which was before my big refactor to use the istanbul libs directly. So these changes are from all that

**Pull request merged**